adam-core/pom.xml

```diff
@@ -181,6 +181,7 @@
     <dependency>
       <groupId>org.seqdoop</groupId>
       <artifactId>hadoop-bam</artifactId>
+      <version>7.4.0</version>
```

Comment:
We'll probably do the update to Hadoop-BAM and HTSJDK in a separate pull request after taking a little time to solicit feedback on the move to JDK8.